### ISSUE TITLE
Hard UX on syncing: Confusing words and no documentation

### ISSUE THREAD

**Issue by amuuza** [nextcloud/android#3075]
Android client has: "Auto upload", "Download", "Sync", "On Device", "Set as available offline"... but no explanation at all, and the behavior of those features are not always the expected one.

File syncing can lead to disasters if the user does not clearly understand what is going to happen with hir files.

As far as I know, Nextcloud Android users would like to understand how to set a folder with:
- One-way sync from server to client
- One-way sync from client to server
- Two-way sync

In case that is possible, what about using these standard and accurate words?

In any case, it would be nice to have some Help, maybe just a very simple Readme file on how each feature works, whatever its name is.

Once that is done, it would also be nice to somehow document how sync conflicts are solved.

**Comment by amuuza** [comment]
In the meantime, as there's no updated documentation, could anyone please help with the following questions?:

Can we have a 2-way sync folder? How?
Can we have a 1-way server-to-client sync folder? How?
Can we have a 1-way client-to-server sync folder? How?
Are there other upload/download/sync options? How to they work?

Thank you!
